fluent/reflect_test.go

```diff
@@ -0,0 +1,95 @@
+package fluent_test
+
+import (
+	"testing"
+
+	. "github.com/warpfork/go-wish"
+
+	ipld "github.com/ipld/go-ipld-prime"
+	"github.com/ipld/go-ipld-prime/fluent"
+	"github.com/ipld/go-ipld-prime/must"
+	basicnode "github.com/ipld/go-ipld-prime/node/basic"
+)
+
+func TestReflect(t *testing.T) {
+	t.Run("ReflectOnMap", func(t *testing.T) {
```

Comment:
nit: this results in TestReflect/ReflectOnMap, so you might as well just do TestReflect/Map

Reply:
fixed.